hw/ip/kmac/dv/cov/kmac_masked_cov_excl.el

```diff
@@ -0,0 +1,84 @@
+INSTANCE:tb.dut.u_tlul_adapter_msgfifo.u_rsp_gen
+CHECKSUM: "2274358225"
+ANNOTATION: "[LOW_RISK] discussed with designer, collecting coverage on adapter SRAM and all internal submodules is considered to be unimportant"
+INSTANCE:tb.dut.u_staterd.u_tlul_adapter.u_rsp_gen
+CHECKSUM: "2024123546"
+ANNOTATION: "[LOW_RISK] discussed with designer, collecting coverage on adapter SRAM and all internal submodules is considered to be unimportant"
+INSTANCE:tb.dut.u_tlul_adapter_msgfifo.u_reqfifo
+CHECKSUM: "2024123546"
+ANNOTATION: "[LOW_RISK] discussed with designer, collecting coverage on adapter SRAM and all internal submodules is considered to be unimportant"
+INSTANCE:tb.dut.u_tlul_adapter_msgfifo.u_sramreqfifo
+CHECKSUM: "2024123546"
+ANNOTATION: "[LOW_RISK] discussed with designer, collecting coverage on adapter SRAM and all internal submodules is considered to be unimportant"
+INSTANCE:tb.dut.u_tlul_adapter_msgfifo.u_rspfifo
+CHECKSUM: "2024123546"
+ANNOTATION: "[LOW_RISK] discussed with designer, collecting coverage on adapter SRAM and all internal submodules is considered to be unimportant"
+INSTANCE:tb.dut.u_staterd.u_tlul_adapter.u_reqfifo
+CHECKSUM: "2024123546"
+ANNOTATION: "[LOW_RISK] discussed with designer, collecting coverage on adapter SRAM and all internal submodules is considered to be unimportant"
+INSTANCE:tb.dut.u_staterd.u_tlul_adapter.u_sramreqfifo
+CHECKSUM: "2024123546"
+ANNOTATION: "[LOW_RISK] discussed with designer, collecting coverage on adapter SRAM and all internal submodules is considered to be unimportant"
+INSTANCE:tb.dut.u_staterd.u_tlul_adapter.u_rspfifo
+CHECKSUM: "1529365951"
+ANNOTATION: "[LOW_RISK] discussed with designer, collecting coverage on adapter SRAM and all internal submodules is considered to be unimportant"
+INSTANCE:tb.dut.u_tlul_adapter_msgfifo
+CHECKSUM: "1529365951"
+ANNOTATION: "[LOW_RISK] discussed with designer, collecting coverage on adapter SRAM and all internal submodules is considered to be unimportant"
+INSTANCE:tb.dut.u_staterd.u_tlul_adapter
+CHECKSUM: "2454292572"
+ANNOTATION: "[LOW_RISK] discussed with designer, collecting coverage on adapter SRAM and all internal submodules is considered to be unimportant"
+INSTANCE:tb.dut.u_tlul_adapter_msgfifo.u_err
+CHECKSUM: "2454292572"
+ANNOTATION: "[LOW_RISK] discussed with designer, collecting coverage on adapter SRAM and all internal submodules is considered to be unimportant"
+INSTANCE:tb.dut.u_staterd.u_tlul_adapter.u_err
+CHECKSUM: "1242460059 1852770110"
+INSTANCE: tb.dut.u_kmac_core
+ANNOTATION: "[UNR] cannot hit due to msgfifo behaviors"
+Block 29 "552024306" "process_latched <= 1'b1;"
+CHECKSUM: "1585181622 4277667638"
+INSTANCE: tb.dut.gen_entropy.u_edn_req
+ANNOTATION: "[UNR] cannot have ack without a req"
+Condition 2 "100884987" "(req_i && ack_o) 1 -1" (1 "01")
+CHECKSUM: "2694308369 151746291"
+INSTANCE: tb.dut
+ANNOTATION: "[UNR] msgfifo cannot be un-empty during Idle state"
```

Comment:
Interesting, so the RTL is redundant? Drop the && and one of the terms?

Reply:
@eunchan can probably provide more details than I can on this one

Reply:
The RTL has been changed over the time. Current design now drops all incoming requests to the Message FIFO until the SW issues the **Start** command or one of the Apps sends valid data.